fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -2250,8 +2241,8 @@ struct ShardedRocksDBKeyValueStore : IKeyValueStore {
 			rocksdb::WriteBatch writeBatch;
 			if (format == DataMoveRocksCF) {
 				CheckpointMetaData& checkpoint = a.checkpoints.front();
-				std::sort(a.ranges.begin(), a.ranges.end(), RangeLessThan());
-				std::sort(checkpoint.ranges.begin(), checkpoint.ranges.end(), RangeLessThan());
+				std::sort(a.ranges.begin(), a.ranges.end(), KeyRangeRef::ArbitraryOrder());
```

Comment:
Why arbirary order?

Reply:
It is an existing Comparator, the order is based on the `KeyRangeRef::begin`, then `KeyRangeRef::end`, not really arbitrary...